### PR TITLE
optimization of token_req_handler

### PR DESCRIPTION
**Pull request by cam-parra**
Signed-off-by: Cam Parra <caeparra@gmail.com>

### CODE REVIEW

File: sovtoken/sovtoken/token_req_handler.py [hunk 1]
```diff
@@ -150,28 +150,32 @@ def _reqToTxn(self, req: Request):
...
+        [self._spend_input(inp["address"], inp["seqNo"],
+                           is_committed=is_committed) for inp in payload[INPUTS]]
+
+        [self._add_new_output(Output(output["address"], seq_no, output["amount"]),
+                              is_committed=is_committed) for output in payload[OUTPUTS]]
 
     def updateState(self, txns, isCommitted=False):
         try:
+            _update_state_mint_public_txn = self._update_state_mint_public_txn
+            _update_state_xfer_public = self._update_state_xfer_public
```
Comment (dastardlychimp) [inline sovtoken/sovtoken/token_req_handler.py]: I don't understand this. Is this to shave off a few chars when the command is used?

File: sovtoken/sovtoken/token_req_handler.py [hunk 2]
```diff
@@ -150,28 +150,32 @@ def _reqToTxn(self, req: Request):
...
-            seq_no = get_seq_no(txn)
-            self._add_new_output(Output(output["address"], seq_no, output["amount"]),
-                                 is_committed=is_committed)
+        seq_no = get_seq_no(txn)
+
+        [self._spend_input(inp["address"], inp["seqNo"],
+                           is_committed=is_committed) for inp in payload[INPUTS]]
+
+        [self._add_new_output(Output(output["address"], seq_no, output["amount"]),
+                              is_committed=is_committed) for output in payload[OUTPUTS]]
```
Comment (dastardlychimp) [inline sovtoken/sovtoken/token_req_handler.py]: I know this is how it was previously, but there are constants for "address", "seqNo", and "amount" in `sovtoken/constants.py`.

File: sovtoken/sovtoken/token_req_handler.py [hunk 3]
```diff
@@ -150,28 +150,32 @@ def _reqToTxn(self, req: Request):
...
 
     def updateState(self, txns, isCommitted=False):
```
Comment (dastardlychimp) [inline sovtoken/sovtoken/token_req_handler.py]: Again, this wasn't you, but I'm pointing out: `isCommitted`, and `is_committed`. Uggh.

File: sovtoken/sovtoken/token_req_handler.py [hunk 4]
```diff
@@ -69,7 +69,7 @@ def _validate_xfer_public_txn(request: Request, sum_inputs: int, sum_outputs: in
 
     @staticmethod
     def validate_given_inputs_outputs(inputs_sum, outputs_sum, required_amount, request,
-                                      error_msg_suffix: Optional[str]=None):
+                                      error_msg_suffix: Optional[str] = None):
```
Comment (dastardlychimp) [inline sovtoken/sovtoken/token_req_handler.py]: I think it is pretty standard in python to not have a space for default values, but it is a little weird with the types.

Comment (cam-parra) [reply sovtoken/sovtoken/token_req_handler.py]: In PEP8 it states that defaults should have spaces in between

File: sovtoken/sovtoken/token_req_handler.py (hunk 1 above)
Comment (cam-parra) [reply sovtoken/sovtoken/token_req_handler.py]: https://wiki.python.org/moin/PythonSpeed/PerformanceTips#Avoiding_dots...

File: sovtoken/sovtoken/token_req_handler.py (hunk 2 above)
Comment (brentzundel) [reply sovtoken/sovtoken/token_req_handler.py]: I agree

File: sovtoken/sovtoken/token_req_handler.py (hunk 3 above)
Comment (brentzundel) [reply sovtoken/sovtoken/token_req_handler.py]: I agree

**Pull request closed without merging**